backend/de.metas.async/src/main/java/de/metas/async/api/impl/QueueDAO.java

```diff
@@ -204,12 +206,9 @@ public QueueDAO()
 		if (packageProcessorIds != null)
 		{
 			wc.append(" AND ");
-			wc.append(I_C_Queue_WorkPackage.COLUMNNAME_C_Queue_Block_ID).append(" IN (")
-					.append(" SELECT ").append(I_C_Queue_Block.COLUMNNAME_C_Queue_Block_ID)
-					.append(" FROM ").append(I_C_Queue_Block.Table_Name).append(" b ")
-					.append(" WHERE ")
-					.append("b.").append(I_C_Queue_Block.COLUMNNAME_C_Queue_PackageProcessor_ID).append(" IN ").append(DB.buildSqlList(packageQuery.getPackageProcessorIds(), params))
-					.append(")");
+			wc.append(I_C_Queue_WorkPackage.COLUMNNAME_C_Queue_PackageProcessor_ID)
+					.append(" IN ")
+					.append(DB.buildSqlList(packageQuery.getPackageProcessorIds(), params));
```

Comment:
u scan use `packageProcessorIds` instead of `packageQuery.getPackageProcessorIds()`.

sorry for my OCD :)

Reply:
😄